swablu/roles.py

```diff
@@ -53,32 +53,24 @@ async def scan_roles():
 def get_hack_type_str(hack_type):
     if hack_type == "balance_hack_wip":
         return "Balance Hack (work in progress)"
-    if hack_type == "balance_hack_demo":
```

Comment:
If you remove these options, what happens with hacks that already have this type?